exercise_05/isAdditionGreaterThanFifty.js

```diff
@@ -1,7 +1,21 @@
 /**
- * #5 :: Export JS function "isAdditionGreaterThanFifty" to check a pair of numbers and return true if their sum is 50 or greater than 50
- * input: number (summand1)
- * input: number (summand2)
- * output: boolean
- * validate input: throw TypeError with message "expected number but received <type-of-argument>" if inputs are not numbers
+ * Check a pair of numbers and return true if their sum is 50 or greater than 50.
+ * 
+ * @param {number} summand1 - The first number to add.
+ * @param {number} summand2 - The second number to add.
+ * @returns {boolean} True if the sum is 50 or greater, otherwise false.
+ * @throws {TypeError} Throws if either `summand1` or `summand2` is not a number.
  */
+const isAdditionGreaterThanFifty = function (summand1, summand2) {
+    if (typeof summand1 !== "number") {
+        throw new TypeError(`expected number but received ${typeof summand1}`);
+    }
+
+    if (typeof summand2 !== "number") {
+        throw new TypeError(`expected number but received ${typeof summand2}`);
+    }
+
+    return (summand1 + summand2) >= 50;
```

Comment:
Avoid magic numbers.

```suggestion
    return (summand1 + summand2) >= THRESHOLD;
```

- [ ] Move value to constant.